sdk/translation/azure-ai-translation-text/azure/ai/translation/text/_patch.py

```diff
@@ -0,0 +1,74 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+from typing import Union
+from azure.core.pipeline import PipelineRequest
+from azure.core.pipeline.policies import ( SansIOHTTPPolicy, BearerTokenCredentialPolicy, AzureKeyCredentialPolicy )
+from azure.core.credentials import ( TokenCredential, AzureKeyCredential )
+
+from ._client import TextTranslationClient as ServiceClientGenerated
+
+def patch_sdk():
+    """Do not remove from this file.
+
+    `patch_sdk` is a last resort escape hatch that allows you to do customizations
+    you can't accomplish using the techniques described in
+    https://aka.ms/azsdk/python/dpcodegen/python/customize
+    """
+
+class TranslatorCredential:
+    def __init__(self, key: str, region: str) -> None:
+        self.key = key
+        self.region = region
+
+class TranslatorAuthenticationPolicy(SansIOHTTPPolicy):
+    def __init__(self, credential: TranslatorCredential):
+        self.credential = credential
+
+    def on_request(self, request: PipelineRequest) -> None:
+        request.http_request.headers["Ocp-Apim-Subscription-Key"] = self.credential.key
+        request.http_request.headers["Ocp-Apim-Subscription-Region"] = self.credential.region
+
+def get_translation_endpoint(endpoint):
+    if not endpoint:
+            endpoint = "https://api.cognitive.microsofttranslator.com"
+
+    translator_endpoint: str = ""
+    if "cognitiveservices" in endpoint:
+        translator_endpoint = endpoint + "/translator/text/v3.0"
+    else:
+        translator_endpoint = endpoint
```

Comment:
Our clients can take an `api_version` keyword only argument in the init. For now we can default it to `v3.0`. That way you'll have access to the API version here and won't need to hardcode it.

Reply:
Done. USing the parameter.